nemo/collections/nlp/data/datasets/sgd_dataset/data_processor.py

```diff
@@ -268,7 +298,17 @@ def _get_state_update(self, current_state, prev_state):
         return state_update
 
     def _create_examples_from_turn(
-        self, turn_id, system_utterance, user_utterance, system_frames, user_frames, prev_states, schemas
+        self,
+        turn_id,
+        system_utterance,
+        user_utterance,
+        system_frames,
+        user_frames,
+        prev_states,
+        schemas,
+        agg_sys_states,
+        slot_carryover_candlist,
+        services_switch_counts,
```

Comment:
please update the docstrings with new args

Reply:
done